packages/cli/src/zosfiles/search/ds/DataSets.definition.ts

```diff
@@ -0,0 +1,96 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandDefinition } from "@zowe/imperative";
+import i18nTypings from "../../-strings-/en";
+import { maxConcurrentRequestsMaxValue } from "../../download/Download.options";
+
+// Does not use the import in anticipation of some internationalization work to be done later.
+const strings = (require("../../-strings-/en").default as typeof i18nTypings).SEARCH;
+const dataSetStrings = strings.ACTIONS.DS;
+
+/**
+ * List all data sets and members command definition containing its description, examples and/or options
+ * @type {ICommandDefinition}
+ */
+export const DataSetsDefinition: ICommandDefinition = {
+    name: "data-sets",
+    aliases: ["ds"],
+    summary: dataSetStrings.SUMMARY,
+    description: dataSetStrings.DESCRIPTION,
+    type: "command",
+    handler: __dirname + "/DataSets.handler",
+    profile: {
+        optional: ["zosmf"]
+    },
+    positionals: [
+        {
+            name: "pattern",
+            description: dataSetStrings.POSITIONALS.PATTERN,
+            type: "string",
+            required: true
+        },
+        {
+            name: "search-string",
```

Comment:
Typically I think we use camel case (e.g., `searchString`) for positional names.